src/applications/find-va-forms/sass/find-va-forms.scss

```diff
@@ -0,0 +1,15 @@
+.find-va-forms-table {
```

Comment:
I know we don't like custom CSS, so I'm open to suggestions! This was the easiest way to add a fixed with for our table headers but I could see it being annoying to update the width if we change the column header text.

Reply:
It isn't fixed width, but the [flex](https://design.va.gov/utilities/flexbox#flex) utilities might be able to get the headers onto one line.

Reply:
Totally. I could see us using either `px` or the flex utilities. Unfortunately, we still have the same problem where if the column header text changes we'd still need to update this styling so it fits on 1 line.